network/server/tchannelthrift/cluster/service.go

```diff
@@ -158,7 +158,7 @@ func (s *service) Write(tctx thrift.Context, req *rpc.WriteRequest) error {
 		return tterrors.NewBadRequestError(err)
 	}
 	ts := xtime.FromNormalizedTime(req.Datapoint.Timestamp, d)
-	err = session.Write(req.ID, ts, req.Datapoint.Value, unit, req.Datapoint.Annotation)
```

Comment:
Should probably also add a truncate method to the cluster service that calls truncate from the session.